Sources/Recycling/ListViews/Notifications/ListView/NotificationsListView.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol NotificationsListViewDelegate: NSObjectProtocol {
+    func notificationsListView(_ notificationsListView: NotificationsListView, didSelectItemAtIndexPath indexPath: IndexPath)
+    func notificationsListView(_ notificationsListView: NotificationsListView, willDisplayItemAtIndexPath indexPath: IndexPath)
+    func notificationsListView(_ notificationsListView: NotificationsListView, didScrollInScrollView scrollView: UIScrollView)
+    func notificationsListView(_ notificationsListView: NotificationsListView, didSelectHeaderAtSection section: Int)
+    func notificationsListView(_ notificationsListView: NotificationsListView, didSelectFooterAtSection section: Int)
+}
+
+public protocol NotificationsListViewDataSource: NSObjectProtocol {
+    func numberOfSections(inNotificationsListView notificationsListView: NotificationsListView) -> Int
+    func notificationsListView(_ notificationsListView: NotificationsListView, numberOfItemsInSection section: Int) -> Int
+    func notificationsListView(_ notificationsListView: NotificationsListView, groupModelAtSection section: Int) -> NotificationsGroupListViewModel
+    func notificationsListView(_ notificationsListView: NotificationsListView, modelAtIndexPath indexPath: IndexPath) -> NotificationsListViewModel
+    func notificationsListView(_ notificationsListView: NotificationsListView, loadImageForModel model: NotificationsListViewModel, imageWidth: CGFloat, completion: @escaping ((UIImage?) -> Void))
+    func notificationsListView(_ notificationsListView: NotificationsListView, cancelLoadingImageForModel model: NotificationsListViewModel, imageWidth: CGFloat)
+}
+
+public class NotificationsListView: UIView {
+    public static let estimatedRowHeight: CGFloat = 106.0
+    public static let estimatedSectionHeaderHeight: CGFloat = 60.0
+    public static let estimatedSectionFooterHeight: CGFloat = 70.0
+
+    // MARK: - Internal properties
+
+    // Have the collection view be private so nobody messes with it.
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView(frame: .zero, style: .grouped)
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.backgroundColor = .white
```

Comment:
Not the most important thing, but maybe use the FinniversKit colors? So for this case: `.milk`.